### ISSUE TITLE
New developers!

### ISSUE THREAD

**Issue by artem-zinnatullin** [artem-zinnatullin/android-wail-app#102]
Just an issue for those, who want to participate in WAIL Beta development, ask your questions and so on.

Better to move this thing to gitter.im + project's wiki, but we can keep this issue for some time.

@ilya-murzinov is main developer at the moment, he can be your mentor! I am @artem-zinnatullin — project maintainer

**Comment by ilya-murzinov** [comment]
:+1: 
Happy to help!

**Comment by cmoyer** [comment]
Sorry about the delay, I've been swamped at work. I'll be more than happy to try and help out on the project where time allows between work and personal projects!

**Comment by artem-zinnatullin** [comment]
@ilya-murzinov @cmoyer everything okay?

**Comment by ilya-murzinov** [comment]
@artem-zinnatullin about what? :))

**Comment by artem-zinnatullin** [comment]
@ilya-murzinov @cmoyer contacted you?

**Comment by ilya-murzinov** [comment]
@artem-zinnatullin not yet

**Comment by cmoyer** [comment]
Yeah I've been swamped with stuff at work. I have about 4 different projects coming to an end that are getting deployed so things have been crazy lately. I will probably pull a copy down sometime this week and start looking through the source to get a general idea of how the application works.

**Comment by ilya-murzinov** [comment]
@cmoyer please concentrate on #8 as soon as you get familiar with the project basics.

**Comment by artem-zinnatullin** [comment]
@ilya-murzinov I think it's better to start with easy to do Pull Requests and later switch on #8 (why the hell it's not number one issue? :D)

For example #110

**Comment by joonki34** [comment]
@ilya-murzinov Are you still working on issue #8? I'd love to know what the current status is. I am planning to add more player supports unless the issue is an ongoing one.

**Comment by ilya-murzinov** [comment]
@joonki34, I am currently working on it.
But we will need all our old receivers because #8 will only work with API 19+ and only if user manually gives WAIL permission to read notifications.
So the old mechanism is not going anywhere and we have to support it.

Would be great if you add support to some new players!

**Comment by joonki34** [comment]
@ilya-murzinov Ok, so I was working on adding support to Musixmatch and I got stuck. I added a new class for the player and a few lines in AndroidManifest.xml. Then WAIL showed the currently playing track but it didn't store tracks. I assume that's because Musixmatch doesn't have 'playback' intent action.

However, RdioPlayerReceiver and SpotifyReceiver don't have the playback action but they are still in WAIL's AndroidManifest.xml. Does this mean that WAIL can't store tracks played by those players? I was trying to test this on one of the players but neither of them is available on my country.

**Comment by ilya-murzinov** [comment]
@joonki34, you should debug the WAILService to see what intent it handles and how it handles it.
Specifically, there is ```addTrackToDB``` method, you should find it usages and figure out why it's not getting called in your case.
Hope that helps.
